diff --git a/src/system/rexruntime.def b/src/system/rexruntime.def
--- a/src/system/rexruntime.def
+++ b/src/system/rexruntime.def
@@ -1,4 +1,3 @@
-LIBRARY rexruntime
 EXPORTS
 ??0AchievementToastDialog@ui@rex@@QEAA@PEAVImGuiDrawer@12@PEAVImmediateDrawer@12@PEAVRuntime@2@@Z
 ??0AchievementsOverlayDialog@ui@rex@@QEAA@PEAVImGuiDrawer@12@PEAVImmediateDrawer@12@PEAVRuntime@2@PEAVAchievementManager@system@2@@Z
@@ -476,3 +475,8 @@ vmaCreateImage
 vmaDestroyAllocator
 vmaDestroyImage
 vmaGetAllocationInfo
+?IncrementCounter@perf@rex@@YAXW4CounterId@12@_J@Z
+?WriteCsvFrame@perf@rex@@YAXXZ
+?FlushCsv@perf@rex@@YAXXZ
+?GetFlagByName@cvar@rex@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@V?$basic_string_view@DU?$char_traits@D@std@@@4@@Z
+?GetRegistry@cvar@rex@@YAAEAV?$vector@UFlagEntry@cvar@rex@@V?$allocator@UFlagEntry@cvar@rex@@@std@@@std@@XZ
